mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/ReIndexerPerformer.java

```diff
@@ -218,7 +218,7 @@ public ReIndexerPerformer(MailboxManager mailboxManager,
                 .flatMap(this::createReindexingEntryFromFailure),
             Flux.fromIterable(previousReIndexingFailures.mailboxFailures())
                 .flatMap(mailboxId -> mapper.findMailboxByIdReactive(mailboxId)
```

Comment:
don't we need MAILBOX_CONCURRENCY on this flatmap ?

Reply:
That should come with a rebase on master hopefully

Reply:
Already the case, line 225